app/assets/stylesheets/navigation.scss

```diff
@@ -1,18 +1,28 @@
-$navbar-brand-font-size: 1.5rem !default;
-$side-nav-width: 320px;
+$side-nav-width: 240px;
 
 nav.top-nav {
-  background-color: $background-color;
+  background-color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`